Assets/MRTK/Providers/OpenXR/Scripts/OculusController.cs

```diff
@@ -0,0 +1,119 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using Microsoft.MixedReality.Toolkit.Input;
+using Microsoft.MixedReality.Toolkit.Utilities;
+using Microsoft.MixedReality.Toolkit.XRSDK.Input;
+using Unity.Profiling;
+using UnityEngine;
+using UnityEngine.XR;
+
+namespace Microsoft.MixedReality.Toolkit.XRSDK.OpenXR
+{
+    [MixedRealityController(
+        SupportedControllerType.OculusTouch,
```

Comment:
This probably needs to be serialized into the controller mapping profile, right?

Reply:
Not completely sure what this entails

Reply:
Went through the Interaction Tool, it seems to work fine and report the mappings we wanted!